pkg/agent/openflow/multicast.go

```diff
@@ -140,19 +140,6 @@ func (f *featureMulticast) multicastOutputFlows(cookieID uint64) []binding.Flow
 	return flows
 }
 
-func (f *featureMulticast) multicastSkipIGMPMetricFlows() []binding.Flow {
-	cookieID := f.cookieAllocator.Request(f.category).Raw()
-	flows := make([]binding.Flow, 0, 2)
-	for _, t := range []*Table{MulticastIngressPodMetricTable, MulticastEgressPodMetricTable} {
```

Comment:
It seems you removed the flow in `MulticastIngressPodMetricTable` by mistake, since this one is not related with your change?

 Besides, if ANP is not enabled, will the removed flow in `MulticastEgressPodMetricTable` introduce mistakes in statistics? @liu4480 @ceclinux

Reply:
@wenyingd I think it will count unwanted IGMP report packets if you remove the 'skip flow' in `MulticastEgressPodMetricTable` because when multicast is enabled and ANP is not enabled, `MulticastEgressRuleTable` does not exist and `MatchProtocol(binding.ProtocolIGMP).Action().GotoStage(stageRouting)...` does not exist neither. Therefore the 'skip flow' in `MulticastEgressPodMetricTable` is still needed.